portworx/README.md

```diff
@@ -0,0 +1,122 @@
+# Terraform Module to install Portworx
+
+This Terraform Module installs the **Portworx Service** on an Openshift (ROKS) cluster on IBM Cloud.
+
+**Module Source**: `git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//portworx`
+
+- [Terraform Module to install Portworx](#terraform-module-to-install-cloud-pak-for-multi-cloud-management)
+  - [Set up access to IBM Cloud](#set-up-access-to-ibm-cloud)
+  - [Provisioning this module in a Terraform Script](#provisioning-this-module-in-a-terraform-script)
+    - [Setting up the OpenShift cluster](#setting-up-the-openshift-cluster)
+    - [Installing Portworx Module](#provisioning-the-portworx-module)
+  - [Input Variables](#input-variables)
+  - [Executing the Terraform Script](#executing-the-terraform-script)
+  - [Clean up](#clean-up)
+
+## Set up access to IBM Cloud
+
+If running these modules from your local terminal, you need to set the credentials to access IBM Cloud.
+
+Go [here](../CREDENTIALS.md) for details.
+
+## Provisioning this module in a Terraform Script
+
+In your Terraform code define the `ibm` provisioner block with the `region` and the `generation`, which is **1 for Classic** and **2 for VPC Gen 2**.
+
+```hcl
+provider "ibm" {
+  region     = "us-south"
+}
+```
+
+### Setting up the OpenShift cluster
+
+NOTE: an OpenShift cluster is required to install this Portworx service. This can be an existing cluster or can be provisioned in the Terraform script.
+
+To provision a new cluster, refer [here](https://github.com/ibm-hcbt/terraform-ibm-cloud-pak/tree/main/roks#building-a-new-roks-cluster) for the code to add to your Terraform script.
+
+### Provisioning the Portworx Module
+
+Use a `module` block assigning `source` to `git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//portworx`. Then set the [input variables](#input-variables) required to install the Portworx service.
+
+```hcl
+module "portworx" {
+  source = "git::https://github.com/ibm-hcbt/terraform-ibm-cloud-pak.git//portworx"
+  enable = true
+  ibmcloud_api_key = "api-key"
```

Comment:
should this be listed as ibmcloud_api_key = "<api-key>". 

This shouldn't literally be "api-key", right?